src/tests/ftest/util/osa_utils.py

```diff
@@ -129,3 +180,51 @@ def verify_single_object(self):
                                       "akey {0}".format(akey)))
         self.obj.close()
         self.container.close()
+
+    def ior_thread(self, pool, oclass, api, test, flags, results):
+        """Start threads and wait until all threads are finished.
+
+        Args:
+            pool (object): pool handle
+            oclass (str): IOR object class
+            api (str): IOR api
+            test (list): IOR test sequence
+            flags (str): IOR flags
+            results (queue): queue for returning thread results
+
+        """
+        container_info = {}
+        mpio_util = MpioUtils()
+        if mpio_util.mpich_installed(self.hostlist_clients) is False:
+            self.fail("Exiting Test : Mpich not installed on :"
+                      " {}".format(self.hostfile_clients[0]))
+        self.pool = pool
+        # Define the arguments for the ior_runner_thread method
+        ior_cmd = IorCommand()
```

Comment:
Can we use the self.ior_cmd param provided by IorTestBase?

Reply:
No... I used this based on Samir's early suggestion that was not available via ior_cmd param. The skeleton of the code is same... Don't want to mess it now. Maybe later.